packages/react/src/radioGroup/RadioGroup.css.ts

```diff
@@ -0,0 +1,71 @@
+import { theme } from "../styles";
+import { style } from "../vanilla-extract";
+
+export const option = style({
+  alignItems: "flex-start",
+  flexDirection: "row",
+  gap: "8px",
+});
+
+export const item = style({
+  all: "unset",
```

Comment:
@toha993 this is setting the `box-sizing` to `content-box`. That is why you were seeing weird sizing issue (border being calculated outside the width/height).

We shouldn't do this. `box-sizing` should remain set to `border-box`.

Reply:
Got it vai.